src/invoices/InvoiceForm/InvoiceForm.js

```diff
@@ -307,6 +310,12 @@ class InvoiceForm extends Component {
                             vertical
                           />
                         </Col>
+                        <Col data-test-col-batch-group xs={3}>
+                          <FieldBatchGroup
```

Comment:
also it would be great to apply the same technique for form, like in https://github.com/folio-org/ui-finance/tree/master/src/common/FiscalYearField